<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageProductList</class>
 <widget class="QWidget" name="StageProductList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ItemTopMenu" name="vTitle" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="wBody" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>602</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#wBody{background-color: rgb(51, 51, 51);}</string>
   </property>
   <widget class="QWidget" name="wSubMenu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#wSubMenu{border-image: url(:/mask/res/images/mask/mask1.png);}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>15</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="3">
      <widget class="ItemChangeBtn" name="btnChangeMode" native="true">
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="ItemBorderLabel" name="wClass" native="true">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lbModeName">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgba(255, 255, 255);
background-color: rgb(255, 255, 255,0);</string>
       </property>
       <property name="text">
        <string>瀏覽方式:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemTopMenu</class>
   <extends>QWidget</extends>
   <header>ItemTopMenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemBorderLabel</class>
   <extends>QWidget</extends>
   <header>ItemBorderLabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemChangeBtn</class>
   <extends>QWidget</extends>
   <header>ItemChangeBtn.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
